### PR TITLE
Benefits of Speaking at DjangoCon

### PR DESCRIPTION
**Pull request by jcjudkins**
-Added to contributors list
-Created a new post. Benefits of Speaking at Djangocon.

### CODE REVIEW

File: _posts/2023-05-08-benefits-of-speaking-djangocon.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- **Professional photographer on hand to photograph your talk.** (Also optional.)
+
+# Still on the fence deciding if you want to give a talk or not?
+
+- **Building your reputation:** Presenting at DjangoCon can establish you as an expert in the Django community, raising your profile and building your reputation.
+- **Networking:** Speaking at DjangoCon provides an opportunity to connect with other developers, designers, and enthusiasts in the Django community, allowing you to expand your network and make new connections.
+- **Career advancement:** Speaking at DjangoCon can help you advance your career by demonstrating your knowledge and expertise, which can lead to new job opportunities or promotions.
+- **Feedback and learning:** Presenting at DjangoCon gives you the opportunity to receive feedback from your peers, which can help you improve your skills and knowledge.
+- **Giving back to the community:** By sharing your knowledge and experience with others at DjangoCon, you are contributing to the growth and development of the Django community, which can be a rewarding experience in itself.
```
Comment (tim-schilling) [inline _posts/2023-05-08-benefits-of-speaking-djangocon.md:30]: > which can be a rewarding experience in itself.

I'd suggest going with "which is a rewarding experience in itself."

It's less tentative and more assertive. If you agree, it's also worth adjusting the "Feedback and learning" and "Travel and conference experience" sections.

File: _posts/2023-05-08-benefits-of-speaking-djangocon.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- **Travel and conference experience:** Speaking at DjangoCon often includes travel to interesting locations and the chance to attend the conference and learn from other presenters, giving you a valuable conference experience that can help broaden your horizons.
+
+## Need Topic Ideas?
+
+Head over to our topic suggestion page! [Topic Suggestions](https://2023.djangocon.us/news/topic-suggestions-for-cfp/)
```
Comment (tim-schilling) [inline _posts/2023-05-08-benefits-of-speaking-djangocon.md:35]: I can't find an example of us doing this similarly, but I'd be curious if `{% post_url _posts/topic-suggestions-for-cfp %}` would work in place of this. It's a jekyll built-in, but I can't test it right now. The idea being that Jekyll will error if that URL/post changes.

File: _posts/2023-05-08-benefits-of-speaking-djangocon.md (hunk 1 above)
Comment (tim-schilling) [reply _posts/2023-05-08-benefits-of-speaking-djangocon.md:30]: To be clear, I'd leave "Build your reputation" and "Career advancement" as they are.

Comment (jcjudkins) [reply _posts/2023-05-08-benefits-of-speaking-djangocon.md:30]: I agree and will make some adjustments! Thank you.

File: _posts/2023-05-08-benefits-of-speaking-djangocon.md (hunk 2 above)
Comment (jcjudkins) [reply _posts/2023-05-08-benefits-of-speaking-djangocon.md:35]: Updated it and it is working great!

**Review by tim-schilling — APPROVED** [review]
Looks good Jason, well done!

Let's get one more person to review it (unless Drew already did) and then we'll merge it.

**Review by PeterGrand — APPROVED** [review]

**Pull request merged**